MekHQ/src/mekhq/gui/PlanetarySystemMapPanel.java

```diff
@@ -0,0 +1,704 @@
+                                Stroke dashed = new BasicStroke(3, BasicStroke.CAP_BUTT, BasicStroke.JOIN_BEVEL, 0, new float[]{9}, 0);
+                                g2.setStroke(dashed);
+                                g2.drawLine(lineX1, lineY1, lineX2, lineY2);
+                            }
+                            if(campaign.getLocation().isAtJumpPoint()) {
+                                //draw a ring around jumpship
+                                drawRing(g2, jumpshipX+(shipImgSize/2), jumpshipY+(shipImgSize/2), shipImgSize/2, Color.ORANGE);
+                                if(null != imgJumpshipFleet) {
+                                    drawRotatedImage(g2, imgJumpshipFleet, 90, jumpshipX, jumpshipY, shipImgSize, shipImgSize);
+                                }
+                            } else if(campaign.getLocation().isOnPlanet()) {
+                                drawRing(g2, x, y, radius, Color.ORANGE);
+                                if(null != imgDropshipFleet) {
+                                    g2.drawImage(imgDropshipFleet, x-radius-shipImgSize, y-radius-shipImgSize, shipImgSize, shipImgSize, null);
+                                }
+                                //draw jumpship too
+                                if(null != imgJumpshipFleet) {
+                                    drawRotatedImage(g2, imgJumpshipFleet, 90, jumpshipX, jumpshipY, shipImgSize, shipImgSize);
+                                }
+                            } else { 
+                                if(null != imgDropshipFleet) {
+                                    int lengthX = lineX1-lineX2;
+                                    int lengthY = lineY1-lineY2;
+                                    double rotationRequired = getFlightRotation(lengthX, lengthY, null != jp && jp.getLastSystem().equals(system), campaign.getLocation().isJumpZenith());
+                                    int partialX = lineX2 + (int) Math.round((lengthX)*campaign.getLocation().getPercentageTransit());
+                                    int partialY = lineY2 + (int) Math.round((lengthY)*campaign.getLocation().getPercentageTransit());
+                                    drawRing(g2, partialX, partialY, shipImgSize/2, Color.ORANGE);
+                                    drawRotatedImage(g2, imgDropshipFleet, rotationRequired, partialX-(shipImgSize/2), partialY-(shipImgSize/2), shipImgSize, shipImgSize);
+                                }
+                                //draw jumpship too
+                                if(null != imgJumpshipFleet) {
+                                    drawRotatedImage(g2, imgJumpshipFleet, 90, jumpshipX, jumpshipY, shipImgSize, shipImgSize);
+                                }
+                            }
+                        }
+                        
+                        //add ring for selected planet
+                        if(i > 0 & selectedPlanet==i) {
```

Comment:
Per LGTM
```suggestion
                        if(selectedPlanet==i) {
```

Reply:
fixed. The for loop now starts at 1 since that is where planets start